test/unit/setup/helpers.js

```diff
@@ -15,6 +15,14 @@ var clearFixtures = function () {
   $body.empty();
 };
 
+before(function() {
+  this.checkProperties = function(block, blacklist) {
+    blacklist = blacklist ? blacklist.push('cid') : 'cid'
+    var props = _.partial(_.omit, _, blacklist);
+    block.call(this, props);
+  }
```

Comment:
I think i'm cool with this, but I'd be curious to see what other @marionettejs/marionette-core think?